### PR TITLE
Remove Naming question, update TOC

### PR DESCRIPTION
**Pull request by fivedots**
This PR removes the naming open question and updates the table of contents. Lately there hasn't been an active discussion on naming, nor consensus on a better scheme, so it seems like the right time to remove it in preparation for shipment.

### CODE REVIEW

File: AccessHandle.md [hunk 1]
```diff
@@ -22,7 +22,7 @@
   - [New data access surface](#new-data-access-surface)
   - [Locking semantics](#locking-semantics)
 - [Open Questions](#open-questions)
-  - [Naming](#naming)
+  - [Exposing AccessHandles on all filesystems](#exposing-accesshandles-on-all-filesystems)
```
Comment (rstz) [inline AccessHandle.md:25]: Not really part of this PR, but you might consider finding a better title here. It's not clear what "filesystems" refers to in this context. Did you mean "file handles"?

File: AccessHandle.md [hunk 2]
```diff
@@ -184,12 +184,6 @@ observe changes done through the new API, even if a lock is still being held.
 
 ## Open Questions
 
-### Naming
-
-The exact name of the new methods hasn’t been defined. The current placeholder
-for data access is *createAccessHandle()* and *createSyncAccessHandle()*.
-*createUnflushedStreams()* and *createDuplexStream()* have been suggested.
```
Comment (rstz) [inline AccessHandle.md:191]: I'm surprised that the names for the streams methods have been decided, can't this wait until they have been implemented?

Comment (rstz) [reply AccessHandle.md:191]: Scratch that, I misunderstood the naming section.

**Review by rstz — APPROVED** [review]

**Review by saleenbefd-wq — COMMENTED** [review]
@hamm
